pkg/frontend/session.go

```diff
@@ -1074,7 +1074,10 @@ func (tcc *TxnCompilerContext) Cost(obj *plan2.ObjectRef, e *plan2.Expr) *plan2.
 	if err != nil {
 		return nil
 	}
-	rows := table.Rows()
+	rows, err := table.Rows(context.TODO())
+	if err != nil {
+		return nil
```

Comment:
return an empty plan2.Cost{} instead of nil that may lead to the planner's error